examples/examples/ts/scatter-aggregate-bar.ts

```diff
@@ -171,11 +171,14 @@ const barOption: echarts.EChartsOption = {
       data: [
         {
           value: calculateAverage(maleDeta, 0),
-          groupId: 'male'
+          groupId: 'female',
```

Comment:
I think the `groupId` should not be changed, and you don't have to add `itemStyle.color`. Instead, we could use `colorBy: 'data'`.